README-dev.md

```diff
@@ -0,0 +1,49 @@
+# How to develop SnarkyJS
```

Comment:
@mitschabaude  I wasn't clear on our title here... what does it mean to "develop SnarkyJS" 
(do we mean to create a package? teach me!)